release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1110,6 +1152,7 @@ This script removes unauthenticated subjects from the following cluster role bin
 
 * The ability to delete operands from the web console using the `delete all operands` option is currently disabled. It will be re-enabled in a future version of {product-title}. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=2012120[BZ#2012120] and link:https://bugzilla.redhat.com/show_bug.cgi?id=2012971[BZ#2012971].
 
+* For clusters that run on {rh-openstack}, a bug in ovn-octavia-provider can cause load balancer listeners to be stuck in a `PENDING_UPDATE` state while the load balancer that they are attached to remains in an `ACTIVE` state. As a result, kuryr-controller can crash. To resolve this problem, update {rh-openstack} to a version that includes a fix as soon as possible. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2019980[*BZ#20004542*])
```

Comment:
Kuryr is just one of the SDN options and not even the default one, so it's worth to underline that it's only affecting these clusters. Let's make sure not to scare anyone without purpose. ;)

Reply:
> Kuryr is just one of the SDN options and not even the default one, so it's worth to underline that it's only affecting these clusters. Let's make sure not to scare anyone without purpose. ;)

```suggestion
* For clusters that run on {rh-openstack} and use Kuryr, a bug in ovn-octavia-provider can cause load balancer listeners to be stuck in a `PENDING_UPDATE` state while the load balancer that they are attached to remains in an `ACTIVE` state. As a result, kuryr-controller can crash. To resolve this problem, update {rh-openstack} to a version that includes a fix as soon as possible. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2019980[*BZ#20004542*])
```

How's that? @dulek

Reply:
Yeah, that's what I mean, thanks!